README.md

```diff
@@ -0,0 +1,40 @@
+### Documentation is included in the Documentation folder ###
+
+[REFrameWork Documentation](https://github.com/UiPath/ReFrameWork/blob/master/Documentation/REFramework%20documentation.pdf)
```

Comment:
In process, the SetTransactionStatus workflows have missing arguments

Reply:
Added value to arguments that were missing it.